src/api/mocks/handlers.ts

```diff
@@ -48,7 +48,7 @@ const usersHandler = http.get<PathParams, DefaultBodyType, GetUsersResponse>('/u
     return HttpResponse.json({ users: allUsers }, { status: 200 });
   }
 
-  const start = page * count;
+  const start = (page - 1) * count;
```

Comment:
why was this change needed after upgrading the router? It looks like the the router has an impact on the api

Reply:
While changing the router I did set up default value for page to 1. Later on I discovered that before it was set to "0" but at least for me starting pagination with 1 makes more sense and might be more common(?)

Reply:
I agree 👍